client/asset/btc/btc.go

```diff
@@ -2047,7 +2047,7 @@ func fund(utxos []*compositeUTXO, enough func(uint64, uint64) bool) (
 	if !tryUTXOs(1) {
 		if !tryUTXOs(0) {
 			return 0, 0, nil, nil, nil, nil, fmt.Errorf("not enough to cover requested funds. "+
-				"%s BTC available in %d UTXOs", amount(sum), len(coins))
+				"%s available in %d UTXOs", amount(sum), len(coins))
```

Comment:
This method is used by all the clones and it's not aware of the asset symbol.  It could be passed down, but it gets kind of ridiculous (tried it), and ultimately the top level caller knows the asset.

Related, which needs to be addressed because of how these errors are wrapped by the callers: https://github.com/decred/dcrdex/issues/1488#issuecomment-1153432115